content/docs/authorization/cli.md

```diff
@@ -891,32 +754,29 @@ Updates a registered storage system.
 karavictl storage update [flags]
 ```
 
-##### Options
-
+##### Required Flags
 ```
-  -e, --endpoint string    Endpoint of REST API gateway
-  -h, --help               Help for update
-  -a, --array-insecure     Array insecure skip verify
-  -p, --pass string        Password (default "****")
-  -s, --system-id string   System identifier (default "systemid")
-  -t, --type string        Type of storage system ("powerflex", "powermax")
-  -u, --user string        Username (default "admin")
+  -e, --endpoint         Endpoint of REST API gateway
+  -a, --array-insecure   Array insecure skip verify
```

Comment:
Should the description of this flag also change to `Skip certificate validation of the storage array`?

Reply:
Yes, updated.